hr_holidays_validity_date/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "HR holidays validity date",
+
+    'summary': """
+        Allow to define start and end date on holidays type.""",
+    'author': 'ACSONE SA/NV,'
+              'Odoo Community Association (OCA)',
+    'website': "https://github.com/OCA/hr",
+    'category': 'Human Resources',
+    'version': '12.0.2.0.0',
```

Comment:
```suggestion
    'version': '12.0.1.0.0',
```
First version in V12 should be `12.0.1.0.0`

Reply:
Changed!